app/controllers/issues_controller.rb

```diff
@@ -15,9 +16,10 @@ class IssuesController < AuthenticatedController
   before_action :set_or_initialize_tags, except: [:destroy]
   before_action :set_auto_save_key, only: [:new, :create, :edit, :update]
 
+  EXTRA_COLUMNS = ['Title', 'Tags', 'Affected', 'Created', 'Created by', 'Updated'].freeze
+
   def index
-    @columns = @issues.map(&:fields).map(&:keys).uniq.flatten |
-      ['Title', 'Tags', 'Affected', 'Created', 'Created by', 'Updated']
+    @columns = collection_field_names(@unsorted_issues) | EXTRA_COLUMNS
```

Comment:
This change clashes with @aapomm's RTP column change. I think you two need to sync on what the fallback strategy is.